### PR TITLE
Setup database connection

### LINKED ISSUE #36: Create a database configuration file
We need a connection file to database that works for both production and local dev environment.

### PR DESCRIPTION
**Pull request by Prabhakar-Poudel**
Added dotenv dependency
Added mongoose dependency
created db connection configuration

closes: #36

### CODE REVIEW

File: server/src/config/database.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable class-methods-use-this */
+import mongoose from 'mongoose';
+
+class Database {
```
Comment (nimish-gupta) [inline server/src/config/database.js]: why class? when it is just an object?

File: server/src/index.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 import express from 'express';
+import 'dotenv/config';
 import cors from 'cors';
+import db from './config/database';
+
+db.connectDb();
 
 const app = express();
 const port = process.env.PORT || 3001;
```
Comment (nimish-gupta) [inline server/src/index.js]: Create a constant file for all process.envs. there you  can also set the defaults

**Review by nimish-gupta — CHANGES_REQUESTED** [review]
LGTM after changes

File: server/src/config/database.js (hunk 1 above)
Comment (Prabhakar-Poudel) [reply server/src/config/database.js]: Makes sense. I started off with different thought and it ended up being a single object.
Will fix.

File: server/src/index.js (hunk 2 above)
Comment (Prabhakar-Poudel) [reply server/src/index.js]: Done.

File: server/src/config/database.js (hunk 1 above)
Comment (Prabhakar-Poudel) [reply server/src/config/database.js]: Done.

**Pull request merged**